lib/responses.js

```diff
@@ -166,6 +168,9 @@ function methodNotAllowed(options) {
     } else {
       code = 405;
       message = 'Method ' + method + ' not allowed';
+
+      log('error', message, { code });
+
```

Comment:
We've been working toward reducing the severity of client errors in the logs.
I think in this case the request logs at the CDN/proxy/load balancer will cover all the context needed, there isn't much more context available here.

Reply:
this i agree we can remove